__tests__/index.js

```diff
@@ -25,6 +25,7 @@ describe('jest-date-mock', () => {
 
     // instanceof
     expect(new Date()).toBeInstanceOf(Date);
+    expect(new Date().constructor(10000)).toBeInstanceOf(Date);
```

Comment:
please help to add more case.

Reply:
I've added some more test cases that ensure that the `.constructor()` call behaves how we'd expect.